lib/utils/transpile-es6.js

```diff
@@ -22,6 +22,10 @@ module.exports = function(tree, description, options) {
   }
 
   var outputTree = map(tree, '**/*.js', function(content, relativePath) {
+    if (relativePath.match(/umd\.js$/)) {
+      return '';
+    }
```

Comment:
i wonder if it is easier to just change the filename to `foo.umd` and then update stuff like: https://github.com/tildeio/rsvp.js/blob/master/Brocfile.js#L29 to point to `foo.umd` instead of `foo.umd.js` then we don't need to filter it out at all, as it wont be a .js file